Sources/Rendering/Misc/SynchronizableRenderWindow/vtkObjectManager.js

```diff
@@ -260,21 +260,26 @@ function genericUpdater(instance, state, context) {
         })
         .finally(context.end); // -> end(arrays)
     });
+    dependencies.push(promises);
```

Comment:
This is wrong, you do not want to have `promises = [[promise, ...]]` but you want to have `promises = [promise, ...]`

Reply:
But since you push your `promise.all().then()` later on, you don't need to add anything here.